### PR TITLE
Feature/AUM daily change DAO-777

### PR DESCRIPTION
**Pull request by Fabricevladimir**
- adds 24h time period indicator on finance page
- changes subtitle (value changed over period of time) text color based on whether change is positive
- bolds `Badge` text

### CODE REVIEW

File: packages/web-app/src/components/wrappers/pageWrappers.tsx [hunk 1]
```diff
@@ -12,9 +12,16 @@ import {useNavigate} from 'react-router-dom';
 export type PageWrapperProps = SectionWrapperProps & {
   buttonLabel: string;
   subtitle: string;
+  timePeriod?: string;
+  sign?: number;
   onClick?: () => void;
 };
 
+const textColors: {[key: string]: string} = {
```
Comment (WissenIstNacht) [inline packages/web-app/src/components/wrappers/pageWrappers.tsx]: To take full advantage of the type system you could do 

```
const textColors: Record<-1 | 0 | 1, string>` = {
  [0]: 'text-ui-600',
  [1]: 'text-success-600',
  [-1]: 'text-ui-600',
};
```

That would have the added benefit that you wouldn't need to translate the sign to a string in `PageWrapper`. On the other hand you'd have to narrow `sign`'s type to `-1 | 0 | 1`.

**Review by WissenIstNacht — APPROVED** [review]
Good to go, thanks! :)

File: packages/web-app/src/components/wrappers/pageWrappers.tsx (hunk 1 above)
Comment (Fabricevladimir) [reply packages/web-app/src/components/wrappers/pageWrappers.tsx]: Ah, yes. Thanks!

**Review by RakeshUP — APPROVED** [review]

**Review by josemarinas — APPROVED** [review]
Seems fine :)

**Pull request merged**